knowledge-base/visual-studio-preview-is-not-working-when-using-objectdatasource.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Report Designer Preview is not working when using ObjectDataSource
```

Comment:
Report Designer Preview is not working when using ObjectDataSource **inside Visual Studio**